### PR TITLE
Add github actions and fix some withdraw tests

### PR DESCRIPTION
**Pull request by nepoche**

### CODE REVIEW

File: test/helpers.js [hunk 1]
```diff
@@ -141,6 +141,30 @@ async function groth16ExportSolidityCallData(proof, pub) {
...
+    pi_b[1][1],
+    pi_c[0],
+    pi_c[1],
+  ]
+  .map(elt => elt.substr(2))
+  .join('');
+
+  return proofEncoded;
+}
+
```
Comment (nepoche) [inline test/helpers.js:167]: Not sure why this function does not work properly - is the verifier.verifyProof modifying the parameters?

Comment (KeltonMad) [reply test/helpers.js:167]: withdraw is failing only after you use this helper function to generate this data?

File: .github/workflows/node.js.yml [hunk 2]
```diff
@@ -16,16 +16,16 @@ jobs:
...
-    - run: npm run build --if-present
-    - run: npm test
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'yarn'
+      - run: yarn install --production=false
+      - run: yarn build
```
Comment (drewstone) [inline .github/workflows/node.js.yml:30]: Consider removing this

**Comment by drewstone** [comment]
Is this ready to review @nepoche ?

**Pull request merged**